pkgs/applications/science/math/maxima/default.nix

```diff
@@ -11,15 +11,17 @@ let
       (stdenv.lib.filter (x: x != null) [ sbcl ecl rlwrap tk gnuplot ]);
 in
 stdenv.mkDerivation ({
+  inherit version;
```

Comment:
I'm not sure what that `version` attribute is for. Nix doesn't use it, and the derivation doesn't reference it either, so this seems superfluous.

Reply:
I use it in another package to locate the fas file:

```
MAXIMA_FAS="${maxima-ecl}/lib/maxima/${maxima-ecl.version}/binary-ecl/maxima.fas";
```

Also I think its a nice de-facto standard to seperate version and name metadata, even if nix doesn't use it.

Reply:
Is there any disadvantage to having the version field exposed? In my opinion its cleaner than first putting it together in the name field and then parsing it again, but even if it isn't cleaner there is no harm right?

Reply:
Isn't `maxima/${maxima-ecl.version}` the same thing as `${maxima-ecl.name}`? I don't see why that construct requires a separate `version` attribute.

Reply:
Ah, crap, now I see it. Sorry, I wasn't thinking straight there for a second.

Reply:
Not precisely the same: `/` vs `-`